website/src/data/users.tsx

```diff
@@ -363,6 +363,86 @@ const Users: User[] = [
+    description:
+      'This sample shows how to take text documents as a input via BlobTrigger, does Text Summarization processing using the AI Congnitive Language service, and then outputs to another text document using BlobOutput binding.',
+      preview: require('./images/test.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/function-csharp-ai-textsummarize',
+    tags: ['functions', 'cognitivesearch', 'dotnetCsharp']
+  },
+  {
+    title: 'Azure Functions - Text Summarization using AI Cognitive Language Service (Python v2 Function)',
+    description:
+      'This sample shows how to take text documents as a input via BlobTrigger, does Text Summarization processing using the AI Congnitive Language service, and then outputs to another text document using BlobOutput binding.',
+      preview: require('./images/test.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/function-python-ai-textsummarize',
+    tags: ['functions', 'cognitivesearch', 'python']
+  },
+  {
+    title: 'Azure Functions - Chat using ChatGPT (Node.js JavaScript Function)',
+    description:
+      'This sample shows how to take a ChatGPT prompt as HTTP Get or Post input, calculates the completions using OpenAI ChatGPT service, and then returns the output plus caches in a Blob state store.',
+      preview: require('./images/test.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/function-javascript-ai-openai-chatgpt',
+    tags: ['functions', 'openai', 'chatgpt', 'javascript']
+  },
+  {
+    title: 'Azure Functions - Chat using ChatGPT (.NET C# v4 Isolated Function)',
+    description:
+      'This sample shows how to take a ChatGPT prompt as HTTP Get or Post input, calculates the completions using OpenAI ChatGPT service, and then returns the output plus caches in a Blob state store.',
+      preview: require('./images/test.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/function-csharp-ai-openai-chatgpt',
+    tags: ['functions','openai', 'chatgpt', 'dotnetCsharp']
+  },
+  {
+    title: 'Azure Functions - Chat using ChatGPT (Python v2 Function)',
+    description:
+      'This sample shows how to take a ChatGPT prompt as HTTP Get or Post input, calculates the completions using OpenAI ChatGPT service, and then returns the output plus caches in a Blob state store.',
+      preview: require('./images/test.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Paul Yuknewicz',
+    source: 'https://github.com/Azure-Samples/function-python-ai-openai-chatgpt',
+    tags: ['functions','openai', 'chatgpt', 'python']
```

Comment:
Might want to also add 'ai' to each of these so it shows up under the general AI tag.

Reply:
Done